rununittests.bat

```diff
@@ -7,4 +7,4 @@ set unitTestsPath=%here%\tests\unit
 set testOutput=%here%\testOutput\unit
 md %testOutput%
 
-call "%scriptsDir%\venvCmd.bat" py -m xmlrunner discover -v -s "%unitTestsPath%" -t "%here%" --output-file "%testOutput%\unitTests.xml" %*
+call "%scriptsDir%\venvCmd.bat" py -m xmlrunner discover -b -s "%unitTestsPath%" -t "%here%" --output-file "%testOutput%\unitTests.xml" %*
```

Comment:
Why the change from -v to -b?

Reply:
the test success output is incredibly noisy, to the point where it cannot be parsed reasonably. `-b` only shows test failures, and makes it easier to parse for both sighted and screen reader devs. the buffer output for screen reader users is F for each test failure, and without supressing punctuation, a "dot" for each test success. 
we use `-v` for appveyor to retain a full log.